src/providers/settings.tsx

```diff
@@ -110,9 +111,11 @@ const getDbText = (apiSettings: any, key: string): any => {
         .replace(/\\n/g, '\n')
         .replace(/(^|[^\n])\n(?!\n)/g, '$1\n\n');
     });
-    return item;
+    return AllHtmlEntities.decode(item);
   } else {
-    return data.replace(/\\n/g, '\n').replace(/(^|[^\n])\n(?!\n)/g, '$1\n\n');
+    return AllHtmlEntities.decode(
+      data.replace(/\\n/g, '\n').replace(/(^|[^\n])\n(?!\n)/g, '$1\n\n')
```

Comment:
What does `.replace(/(^|[^\n])\n(?!\n)/g, '$1\n\n')` do?

Reply:
@floridemai  It was there before, not my code, but I think it cleans up common broken `\n`s from translated text.

I think most if not all the apps have this.